docs/faq/proof-of-stake/pos-general-faq.md

```diff
@@ -0,0 +1,119 @@
+
+For example, you could choose the colour of your ticket to be either red or blue, set that option, and the system will count for the next 1000 blocks how many red tickets and how many blue tickets there are. Maybe there are 3000 red tickets, 1500 blue tickets, and 500 that didn't select a colour.
+
+If you replace the red or blue option with a yes or no option, you have a voting system that can count votes over time. By using the PoS voting system anyone who holds DCR can vote on issues set in an agenda.
+
+This voting system can be used for any issue. For Decred the most prominent use will be to vote on hardforks.
+
+---
+
+#### **8. What is hardfork voting?**
+
+Like any other cryptocurrency Decred might need to hardfork at some point.
+
+One of the agenda issues set for the PoS voting system could be a hardfork. If such an issue is set, the new version of the decred files will include the hardfork but it will not activate until the PoS system has voted on it.
+
+For a hardfork vote to start two critical conditions have to be met:
+
+- First, 75% of the PoW miners have to upgrade to the latest version of the current network block version. This check runs over the past 100 blocks.
+- Second, 75% of the PoS miners have to upgrade to the latest version. This check runs over the past 2016 blocks.
+
+If these checks are met the voting process begins. Tickets can be set with an additional parameter of yes/no/abstain. You do this in your wallet before the ticket votes. Tickets marked as abstain will not be counted in the total amount of votes.
+
+The PoS system then starts counting tickets with these parameters set over a predetermined amount of blocks. If over this interval the amount of votes is 75%, or more, 'yes' the vote has passed. A lock in period of a number of blocks will start before the hardfork goes live, so everyone has a chance to upgrade and not be hardforked off the network.
+
+Because the code for the hardfork is already in the then current version of Decred there is no need for the developers to interfere or the majority of PoW miners and PoS miners to upgrade after a decision has been reached. If the vote on a hardfork passes it will be implemented automatically after the lock in period.
+
+The hardfork vote can fail at multiple points. The vote might not start at all if the PoW miners or PoS miners do not upgrade. After that, the treshhold of 75% votes might not be reached.
+
+In case a vote fails a new round will start at the beginning of the process. This means checking whether the PoW miners have upgraded, if the PoS miners have upgraded, and then another vote counting period. This will continue for a certain amount of rounds, after which, if the vote hasn't passed, the agenda issue will be tabled.
+
+---
+
+## **<i class="fa fa-book"></i> Links**
+
+[^1]: [Paymetheus Ticket Purchasing Guide] (/getting-started/user-guides/using-paymetheus.md#-purchase-tickets-tab)
```

Comment:
You can't have a space between the anchor text and the link if you want it to render properly.  So no space between the ] and the (